### ISSUE TITLE
Make About page into public page for live link for MEC 2019

### ISSUE THREAD

**Issue by emilyhopkins** [ELVIS-Project/simssadb#283]
The page being built in #273 will also be the live page for public viewing that is up for the MEC presentation.

**Comment by emilyhopkins** [comment]
The page is ready!

**Comment by juyaolongpaul** [comment]
Now the about page can be visited at http://db.simssa.ca/.

**Comment by emilyhopkins** [comment]
It looks like absolutely none of the styling from the original site is going over? Frankly it looks pretty bad :(

<img width="1225" alt="Screen Shot 2019-05-27 at 11 33 57 AM" src="https://user-images.githubusercontent.com/13374693/58429277-62482380-8073-11e9-972f-af746efb1d8a.png">

Also the changes you made to the text aren't necessary -- the intro I wrote already includes the fact that it's under development, and it should just say "Welcome to the SIMSSA Database" not "Welcome to the Introduction of the SIMSSA Database"

Please fix this ASAP, the current page looks really amateurish!

**Comment by juyaolongpaul** [comment]
I read your text carefully before conducting any changes, and it talks about the "prototype of the SIMSSA DB", which fits well if the DB can be visited, but in this case it does not (we only have the about page), so that's why I change the title since it is only the introduction of the SIMSSA DB, not the DB itself (again, the DB is not available and cannot be visited). 

The reason why I add the note is that the original page is not perfectly clear that the DB is not available, and it might leave visitors wondering where the DB can be found and visited. I think this will help when I imagine myself as a visitor.

In terms of the style, I have considered incorporating the style and format when I worked on it last Friday, but unfortunately it involves serving a series of CSS files and static files and linking them to the static page. Currently, the page is served on our Nginx router from Compute Canada (since we decided not showing the actual DB on Compute Canada, this is the way to make the about page vistable), and this server is not ideal for serving static files and CSS files since it serves the purpose of routing URLs to individual servers that host our websites. 

I hope this makes it clear that I deliberate all the options before I proceed my work, and they are not rash decisions. 

If the team agrees that (1) Linking a style to the page is vital for this prototype page at this point and (2) we can copy and serve these static files on Nginx router, then I can look into it, but I cannot guarantee it to be done due to my schedule of travelling. Maybe someone else can help in this case.

**Comment by emilyhopkins** [comment]
Hi Yaolong, sorry, I know there's been a lot going on, the current text is just a bit confusing and combined with the lack of styling comes off less than professional. I've suggested an alternative for the text below.

@blueNoteb5 is fixing the styling something you can help with? Not having a demo is one thing but this current under construction page is almost worse than nothing and doesn't really reflect all the work that has been done on the site so far. 

How about this for the starting text instead (heading levels stay the same):

~~Note: SIMSSA Database is currently under construction, but we are diligently working on it, and aim to make an official release as soon as possible once it is ready. Stay tuned!~~

Welcome to ~~the introduction of~~ the SIMSSA Database!

The database is currently under construction – stay tuned for the official release soon!

The SIMSSA Database is designed as a repository and discovery tool for symbolic music files (e.g. MEI, Kern, MusicXML, and MIDI). Users can browse existing files or upload their own. ~~The current site is a prototype that is still under development.~~ It serves as part of the SIMSSA Project, a SSHRC Partnership Grant. The SIMSSA Database is the successor of an older database created as part of Julie Cumming’s Digging into Data grant, designed to gather symbolic music files in one place to do computer-aided counterpoint analysis. The new database has made improvements in several areas, explained below.

(rest of text continues as-is)

**Comment by juyaolongpaul** [comment]
I just changed the text as you suggested. I wish I could have helped more, but this is what I can for now (I need to pack and leave soon).

**Comment by emilyhopkins** [comment]
Thanks & safe travels!

**Comment by codaich** [comment]
**HIGH PRIORITY** (@juyaolongpaul and/or @blueNoteb5 or anyone else with login privileges to the Compute Canada site): It seems like most of the links on http://db.simssa.ca are broken, including the links to the SIMSSA project, the DB GitHub, the DB documentation and more  (I think it may be the incorrect quote symbols in the HTML). This **MUST** be fixed before Yalong's presentations, as it is unprofessional in the extreme.

**Comment by juyaolongpaul** [comment]
Hi Cory, thanks for catching this. I found the cause (I mentioned you there), and it is easy to fix. 
Currently, this page is hosted on the Nginx router from Compute Canada, so anyone (@blueNoteb5,  Alex, or others) in the lab that can access the router can help fix this issue (I can help the person locate the html file and other stuff if needed).

**Comment by vigliensoni** [comment]
There were some curly (“) instead of straight (") quotes in the links that broke the links. I fixed them on the live site (not on the Github repo) and now they seem to be working.

**Comment by vigliensoni** [comment]
@juyaolongpaul  you should add the CSS file in order to have the style. Or, better, clone and serve the full "SIMSSA DB About only repo" https://github.com/ELVIS-Project/SIMSSA-DB-About-only)

**Comment by juyaolongpaul** [comment]
Done! Thanks, Gabriel and Emily for the help.

**Comment by emilyhopkins** [comment]
(you can ignore my other comments about fixing the quotation marks! They were correct in the repo above the whole time, I just hadn't corrected the old page!) 

Thanks for updating the site! @deepio was going to help me do this today so it's great that it's already done.
